packages/terra-menu/src/Menu.jsx

```diff
@@ -168,18 +149,20 @@ class Menu extends React.Component {
         {...customProps}
         boundingRef={boundingRef}
         isArrowDisplayed={isArrowDisplayed}
+        attachmentBehavior="flip"
         contentAttachment={isArrowDisplayed ? 'top center' : 'top right'}
-        contentHeight={popupHeight.toString()}
+        contentHeight="auto"
         contentWidth={this.props.contentWidth}
         classNameArrow={arrowClass}
-        classNameContent={contentClass}
+        classNameContent={classNameContent}
         classNameOverlay={classNameOverlay}
         isOpen={isOpen}
         onRequestClose={onRequestClose}
         targetRef={targetRef}
         isHeaderDisabled
+        isContentFocusDisabled
       >
-        <SlideGroup items={slides} />
+        {slides}
```

Comment:
So in conjunction with my comment above about `setPageDimensions`, maybe rename slides to pages to match context